components/header/Header.tsx

```diff
@@ -0,0 +1,71 @@
+import Banner from "@/components/header/Banner"
+import Navbar from "@/components/header/Navbar"
+import getOpenIssues from "@/components/header/utils"
+
+export default async function Header() {
+  const repositories = await getOpenIssues()
+  // uncomment to test banner with no open issues
```

Comment:
I'm a little confused about the comment vs uncomment for testing here? Might be good to make that a little clearer